### PR TITLE
rename drilldown links to explore in kibana

### PR DESCRIPTION
**Pull request by shalper**
# What changed

----

## Pages to review

- https://deploy-preview-517--logz-docs.netlify.app/user-guide/infrastructure-monitoring/explore-in-kibana-drilldown-links.html

### CODE REVIEW

File: _source/user-guide/infrastructure-monitoring/explore-in-kibana.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: article
-title: Drilldown links
+title: Explore in Kibana
 permalink: /user-guide/infrastructure-monitoring/explore-in-kibana-drilldown-links
 flags:
   logzio-plan: pro
```
Comment (shalper) [inline _source/user-guide/infrastructure-monitoring/explore-in-kibana.md:6]:
```suggestion
  logzio-plan: community
```

File: _source/user-guide/infrastructure-monitoring/explore-in-kibana.md [hunk 2]
```diff
@@ -12,14 +12,12 @@ contributors:
   - daniel-tk
 ---
 
-Drilldown links are shortcuts that take you directly to the relevant Kibana results in your logs.
-When you identify an issue in your graph and want to investigate it further,
-you can click a drilldown link
-and go straight to the related logs.
+**Explore in Kibana** are drilldown links that give you a direct shortcut from your Metrics' visualizations to the relevant logs in Kibana.
+
+You'll find that many of the dashboards provided out-of-the-box by Logz.io come with **Explore in Kibana** drilldown links preconfigured. But you can also add your own to any dashboards and visualizations. (The only caveat is you'll need to have edit-permissions to the dashboard, so if you don't, duplicate the dashboard first.)
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/explore-in-kibana.md:17]: That's not true unfortunately.
We don't have pre-configured explore in kibana links, as they are pretty specific per user. It is mostly something that they need to do own their own (that's why in the guide I wrote there are detailed examples and explanations - so they will have a reference)

File: _source/user-guide/infrastructure-monitoring/explore-in-kibana.md [hunk 3]
```diff
@@ -31,21 +29,19 @@ you can configure your own drilldown links.
 {:.no_toc}
 
 **Before you begin, you'll need**:
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/explore-in-kibana.md:31]: That's not true technically. they don't HAVE to have variables, they can also hard-code the values to the link, although probably no one will ever do that.
So maybe change the phrasing to "we highly recommend on using variables" or something like that...

File: _source/user-guide/infrastructure-monitoring/explore-in-kibana.md [hunk 4]
```diff
@@ -63,30 +59,45 @@ If you plan to add alerts to the visualization, note that only **Graph** is supp
...
-Scroll to the _Drilldown links_ section,
-and click **+ Add link**.
-You'll see the full drilldown link configuration.
+  ```
+  /#/explore-kibana-from-grafana?$__url_time_range&query=
+  ```
+* Write your Kibana query in Lucene syntax. Make sure there are no spaces before and after the `=` operator.
 
-![Drilldown links](https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana/panel-config--general--drilldown-link-config.png)
+  It is a good idea to test your query in Kibana Discover to make sure you're getting the intended results.
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/explore-in-kibana.md:77]: Great sentence

File: _source/user-guide/infrastructure-monitoring/explore-in-kibana.md [hunk 5]
```diff
@@ -63,30 +59,45 @@ If you plan to add alerts to the visualization, note that only **Graph** is supp
...
+
+Here's how to check your variable settings or change them:
+
+<video autoplay loop>
+  <source src="https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana-videos/grafana-variables.mp4" type="video/mp4" />
+</video>
+
+* If **Multi-value** or **Include All option** are toggled on, write the variable name as usual - for example `$node`.
+* If they are toggled off, write the variable name between double quotes - for example `“$node”`.
+
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/explore-in-kibana.md:91]: Great, exactly

File: _source/user-guide/infrastructure-monitoring/explore-in-kibana.md [hunk 6]
```diff
@@ -63,30 +59,45 @@ If you plan to add alerts to the visualization, note that only **Graph** is supp
...
+Suppose you have a **Node** variable filtered on a specific node. If you wish to look for Kubernetes logs related to the specific node your dashboard is currently filtered on, your URL should look like this:
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/explore-in-kibana.md:92]: Change "your URL should like this" to "may/might look like this" cause it's only a suggestion, not a must

File: _source/user-guide/infrastructure-monitoring/explore-in-kibana.md [hunk 7]
```diff
@@ -96,6 +107,6 @@ Save your dashboard.
 Test your new drilldown link
 by hovering over <i class="fas fa-external-link-alt"></i>
 (upper left corner of the panel),
-and then clicking **Explore in Kibana**.
+and clicking **Explore in Kibana**.
 
 ![Drilldown link](https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana/panel-drilldown-link.png)
```
Comment (daniel-tk) [inline _source/user-guide/infrastructure-monitoring/explore-in-kibana.md:112]: Brilliant! Well done!

File: _source/user-guide/infrastructure-monitoring/explore-in-kibana.md (hunk 2 above)
Comment (shalper) [inline _source/user-guide/infrastructure-monitoring/explore-in-kibana.md:17]:
```suggestion
A few dashboards provided out-of-the-box by Logz.io come with **Explore in Kibana** drilldown links preconfigured. But generally speaking, these are very user-specific, so you'll want to set them up for yourself to suit your particular implementation.  
```

**Pull request merged**